src/main/java/com/example/atm/exception/RequestCustomException.java

```diff
@@ -0,0 +1,15 @@
+package com.example.atm.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.BAD_REQUEST)
+public class RequestCustomException extends RuntimeException {
```

Comment:
не дуже симпатична назва помилки) Давай краще так?
```suggestion
public class BadRequestException extends RuntimeException {
```

Reply:
або щось типу такого